modules/raveboard/.gitignore

```diff
@@ -40,6 +40,7 @@ npm-debug.log
 yarn-error.log
 testem.log
 /typings
+package-lock.json
```

Comment:
This we should think about. Usually, package-lock.json is committed to the repo to ensure that you can install dependencies with the exact version (package.json is not too exact). However, if the UI stays a small project, we could ignore the lockfile.

Reply:
I would ignore for now... I would prefer putting exact versions in package.json if needed...